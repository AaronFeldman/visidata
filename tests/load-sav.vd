sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/accidents.sav	o	
