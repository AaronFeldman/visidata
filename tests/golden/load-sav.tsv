agecat	gender	accid	pop
1.00	1.00	57997.00	198522.00
2.00	1.00	57113.00	203200.00
3.00	1.00	54123.00	200744.00
1.00	0.00	63936.00	187791.00
2.00	0.00	64835.00	195714.00
3.00	0.00	66804.00	208239.00
